Sources/OpenTelemetrySdk/Metrics/Stable/Exporter/StablePeriodicMetricReaderSdk.swift

```diff
@@ -22,7 +22,7 @@ public class StablePeriodicMetricReaderSdk : StableMetricReader {
         scheduleTimer = DispatchSource.makeTimerSource(flags: DispatchSource.TimerFlags(), queue: scheduleQueue)
         
         scheduleTimer.setEventHandler { [weak self] in
-            autoreleasepool {
+            maybeAutoreleasepool {
```

Comment:
Not sure how this will effect things. We'd be more comfortable if this was also in an IFDEF